### ISSUE TITLE
issue with @react-native-community/eslint-configcompatibility  3.0.0

### ISSUE THREAD

**Issue by mlecoq** [wcandillon/eslint-plugin-reanimated#28]
Hi,

after upgrading to 3.0.0 of @react-native-community/eslint-config

I have the following error 

```
error An unexpected error occurred: "could not find a copy of typescript to link in /node_modules/eslint-plugin-reanimated/node_modules".
```

Rolling back to 2.0 of eslint-config solves the issue

**Comment by mlecoq** [comment]
I am using Yarn v1.22
